apps/docs/pages/guides/auth/social-login/auth-kakao.mdx

```diff
@@ -0,0 +1,88 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-kakao',
+  title: 'Login with Kakao',
+  description: 'Add Kakao OAuth to your Supabase project',
+}
+
+To enable Kakao Auth for your project, you need to set up an Kakao OAuth application and add the application credentials to your Supabase Dashboard.
+
+## Overview
+
+Kakao OAuth consists of six broad steps:
+
+- Create and configure your app in the [Kakao Developer Portal](https://developers.kakao.com).
+- Obtaining a `REST API key` - this will serve as the `client_id`.
+- Generating the `client_secret` using the `Client secret code`.
+- Add your `client id` and `client secret` keys to your [Supabase Project](https://app.supabase.com).
+- Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js).
+
+## Access your Kakao Developer account
+
+- Go to [Kakao Developers Portal](https://developers.kakao.com).
+- Click on `Login` at the top right to log in.
+
+![Kakao Developers Portal.](/docs/img/guides/auth-kakao/kakao-developers-page.png)
+
+## Create and configure your app
+
+- Go to `My Application`.
+- Click on `Add an application` at the top.
+- Fill out your app information:
+  - App icon.
+  - App name.
+  - Company name.
+  - Click `Save` at the bottom right.
+
+## Obtain a REST API key
+
+This will serve as the `client_id` when you make API calls to authenticate the user.
+
+- Go to `My Application`.
+- Click on your app that you have just created.
+- You will be redirected to `Summary` tab of your app.
+- In the `App Keys` section you will see `REST API key` -- this ID will become your `client_id` later.
+
+## Find your callback URL
```

Comment:
can you also indicate where to add the callback URL on kakao please?

Reply:
We would just need a sentence to tell the user to add the callback url to this section

<img width="1325" alt="image" src="https://github.com/supabase/supabase/assets/28647601/92effa5d-5f54-4b71-a6b3-28ad7183f89e">

Reply:
Yup. Will it be good to put the sentence at the bottom inside `Find your callback URL` section?